package.json

```diff
@@ -16,8 +16,8 @@
     "build:dev": "webpack --mode development --https false",
     "build-dev": "webpack --mode development --https false && echo . && echo . && echo . && echo Please use 'build:dev' instead of 'build-dev'.",
     "dev-server": "webpack-dev-server --mode development",
-    "lint": "eslint **/*.{js,jsx}",
-    "lint:fix": "eslint **/*.{js,jsx} --fix",
+    "lint": "eslint '**/*.{js,jsx}'",
+    "lint:fix": "eslint '**/*.{js,jsx}' --fix",
```

Comment:
Why are these quotes needed?

Reply:
@algomaster99 
If no quotation marks are applied, then [this](https://github.com/eslint/eslint/issues/10880) issue occurs.
So the quotation marks are required for the lint command to run properly.